MeshPipe/MeshPipe.m

```diff
@@ -64,36 +80,39 @@ - (instancetype)initWithBasePort:(int)basePort count:(int)count peerName:(NSStri
 	_potentialPeers = [NSMutableArray new];
 	_peers = [NSMutableSet new];
 	
-	_listenSocket = [[AsyncUdpSocket alloc] initWithDelegate:self];
-
+	_listenSocket = [[GCDAsyncUdpSocket alloc] initWithDelegate:self
+                                                  delegateQueue:dispatch_get_main_queue()];
 	for(int i = 0; i < count; i++) {
 		int port = _basePort + i;
-		BOOL success = [_listenSocket bindToAddress:@"localhost" port:port error:NULL];
+        NSData *address = [MeshPipe _addressFor:MeshPipeLocalAddress port:port];
+		BOOL success = [_listenSocket bindToAddress:address error:NULL];
 		if(success) {
 			_myPort = port;
 			break;
 		}
 	}
 	if(!_myPort) {
-		NSLog(@"No free MeshPipe slots, not creating pipe");
+		MPDebug(@"No free MeshPipe slots, not creating pipe");
```

Comment:
I agree that fatal errors should go to console. I don't have a strong opinion whether `MPLogError` or `NSLog` would be preferred here. I'll add `MPLogError`. :)